js/ui/scheduler/ui.scheduler.timeline.js

```diff
@@ -288,7 +288,7 @@ var SchedulerTimeline = SchedulerWorkSpace.inherit({
             fullDays = Math.floor(fullInterval / (toMs("day"))),
             tailDuration = fullInterval - (fullDays * toMs("day")),
             tailDelta = 0,
-            cellCount = this._getCellCountInDay() * fullDays,
+            cellCount = this._getCellCountInDay() * (fullDays - 2 * this._getWeekendsCount(fullDays)),
```

Comment:
_getIntervalBetween is the base method which call _getWeekendsCount.
_getWeekendsCount method is  redefined for the timelineWorkWeek